packages/pharos/src/components/sheet/pharos-sheet.scss

```diff
@@ -77,6 +77,9 @@
 
   .sheet__wrapper {
     padding: 0 var(--pharos-spacing-one-and-a-half-x);
+    display: flex;
+    flex-direction: column;
+    height: calc(100% - 67px);
```

Comment:
Where does this `67px` value arise from?

Reply:
is specifying the height necessary? doesn't seem to effect it if i remove it
![Screenshot 2023-09-14 at 4 05 08 PM](https://github.com/ithaka/pharos/assets/1250638/578fcfba-cae3-407d-87a5-ca4d9b6194a7)

Reply:
ah nvm. i see it's needed for overflow scrolling 🤔

Reply:
does replacing the `height` with `overflow-y: scroll` achieve the same result?
![Screenshot 2023-09-14 at 4 20 37 PM](https://github.com/ithaka/pharos/assets/1250638/e6edf479-cfaa-4b12-9f03-f6f4bfdc7cfd)

Reply:
Thank you for the advice! After discussing with Lori we now decide to actually remove the footer from the sheet (at least for now) to match the treatment we have in the popover. So this problem naturally solves.